### ISSUE TITLE
[Screen Reader - Adaptive Cards 1.0] Screen reader Voiceover announces Text Field as role information with column which has select action on “Column.SelectAction” page.[Rendering] Bug Title Here

### ISSUE THREAD

**Issue by Shireen2000** [microsoft/AdaptiveCards#7816]
### Target Platforms

iOS

### SDK Version

1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 12
iOS Version: 15.4.1
App: Microsoft Teams
Microsoft Team's version: 4.6.1
Obi Version: 2.0.1942.2

**User Experience:**
Visually impaired users who depend on screen reader will not get to know the correct type of control due to which they will assume a wrong functionality for that control.

**Note:** Login Credentials must not be provided within the bug.

**Repro Steps:**
1. Launch Teams app in iOS.
2. Navigate to Teams option and select it.
3. Navigate to Accessibility under Azure Obi UX and select it.
4. Navigate to 'New Conversation' and select it.
5.Type '@Obi' and send it.
6. Some content appears, navigate to 'IcM' and select it.
7. Navigate to the content shown below.
8. Verify the issue using voiceover.

**Actual Result:**
Voiceover announces incorrect role as text field for all the buttons present under ICM and VSTS. On double tapping on them new messages appear on the screen as they function like buttons.

**Expected Result:**
Voiceover should announce correct role as 'Button" for all the buttons when focus lands on them present under ICM and VSTS as on double tapping on them new messages appear on the screen.

**MAS Reference:**
[MAS 4.1.2 – Name, Role, Value ](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b507B5D90-F1B9-4AE1-91D1-2F1F2D6C7C1D%7d&file=MAS%204.1.2%20%E2%80%93%20Name%2c%20Role%2c%20Value.docx&action=default&mobileredirect=true) 

**Reference Links:**
[Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Screenshots

https://user-images.githubusercontent.com/96283152/188956726-bd61e0e7-3802-4697-8906-ac204d349fae.mov

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
@jwoo-msft Can we transfer/close this issue since we no longer own iOS?

**Comment by vagpt** [comment]
Closing this issue and if the same issue persist then we will log new issue is in new ADO.
